app/src/main/res/layout/detailed_recipe_fragment.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <RelativeLayout
```

Comment:
Relative layout don't have orientation property. And BTW are you sure you didn't want to use LinearLayout here?